controllers/src/main/scala/cromwell/pipeline/controller/utils/ExceptionMapper.scala

```diff
@@ -0,0 +1,38 @@
+package cromwell.pipeline.controller.utils
+
+import akka.http.scaladsl.model.{ StatusCode, StatusCodes }
+import akka.http.scaladsl.server.Directives.complete
+import akka.http.scaladsl.server.ExceptionHandler
+import cromwell.pipeline.service.ProjectService.Exceptions._
+import cromwell.pipeline.service.UserService.Exceptions._
+import cromwell.pipeline.service.{ ProjectService, UserService }
+
+import scala.language.implicitConversions
+
+object ProjectServiceExceptionMapper {
+  implicit def excToStatusCode(e: ProjectServiceException): StatusCode = e match {
+    case _: ProjectService.Exceptions.NotFound     => StatusCodes.NotFound
+    case _: ProjectService.Exceptions.AccessDenied => StatusCodes.Forbidden
+    case _                                         => StatusCodes.InternalServerError
+  }
+
+  implicit val projectServiceExceptionHandler: ExceptionHandler = ExceptionHandler {
+    case e: ProjectServiceException => complete(e, e.getMessage)
```

Comment:
Please, don't use implicits in this way. They should mainly be used for type classes, passing contexts (e.g. execution context) and conversion between related (or even isomorphic) types. But this conversion might not be obvious for some people, so please convert it explicitly 
```suggestion
    case e: ProjectServiceException => complete(toStatusCode(e), e.getMessage)
```
It's not that much code, but is far more readable

Reply:
I'll fix it.

Reply:
Done